tests/foreman/cli/test_import.py

```diff
@@ -539,11 +544,204 @@ def test_import_users_recovery(self, test_data):
+                len(HostCollection.list(
+                    {'organization-id': imp_org['sat6']}
+                ).stdout), 0
+            )
+
+    @data(*import_org_data())
+    def test_reimport_host_collections_default_negative(self, test_data):
+        """@test: Import all System Groups from the default data set
+        (predefined source) as the Host Collections and try to import
+        them again.
+
+        @feature: Repetitive Import Host-Collections
+
+        @assert: 3 Host Collections created
+
+        """
+        files = dict(self.default_dataset[1])
+        files['users'] = update_csv_values(
+            files['users'],
+            test_data,
+            self.default_dataset[0]
+        )
+        files['system-groups'] = update_csv_values(
+            files['system-groups'],
+            test_data,
+            self.default_dataset[0]
+        )
+        # import the prerequisities
+        self.assertEqual(
+            Import.organization({'csv-file': files['users']}).return_code, 0
+        )
+        self.assertEqual(
+            Import.host_collection(
+                {'csv-file': files['system-groups']}
+            ).return_code, 0
+        )
+        transition_data = csv_to_dataset(
+            ssh.command(
+                u'ls ${HOME}/.transition_data/organization*'
+            ).stdout[:-1]
+        )
+        hcollections_before = [HostCollection.list(
+            {'organization-id': tr['sat6']}
+        ).stdout for tr in transition_data]
+        self.assertEqual(
+            Import.host_collection(
+                {'csv-file': files['system-groups']}
+            ).return_code, 0
+        )
+        hcollections_after = [HostCollection.list(
+            {'organization-id': tr['sat6']}
+        ).stdout for tr in transition_data]
+        self.assertEqual(hcollections_before, hcollections_after)
+
+    @data(*import_org_data())
+    def test_import_host_collections_recovery(self, test_data):
+        """@test: Try to Import Collections with the same name to invoke
+        usage of a recovery strategy (rename, map, none)
+
+        @feature: Import HostCollection Recover
+
+        @assert: 2nd Import will rename the new collections, 2nd import will
+        map them and the 3rd one will result in No Action Taken
+
+        """
+        # prepare the data
+        files = dict(self.default_dataset[1])
+        files['users'] = update_csv_values(
+            files['users'],
+            test_data,
+            self.default_dataset[0]
+        )
+        files['system-groups'] = update_csv_values(
+            files['system-groups'],
+            test_data,
+            self.default_dataset[0]
+        )
+        # initial import
+        self.assertEqual(
+            Import.organization({'csv-file': files['users']}).return_code, 0)
+        self.assertEqual(
+            Import.host_collection(
+                {'csv-file': files['system-groups']}
+            ).return_code, 0)
```

Comment:
Place the closing parentheses at the same indentation level as the beginning of the statement.

Use a for loop.